docs/howtos/cert-manager.md

```diff
@@ -187,9 +182,9 @@ The HTTP-01 challenge verifies ownership of the domain by sending a request for
 
 4. Create a Mapping for the `/.well-known/acme-challenge/` route.
 
-cert-manager uses an `Ingress` resource to issue the challenge to `/.well-known/acme-challenge/` but, since Ambassador is not an `Ingress`, we will need to create a `Mapping` so the cert-manager can reach the temporary pod.
+    cert-manager uses an `Ingress` resource to issue the challenge to `/.well-known/acme-challenge/` but, since Ambassador is not an `Ingress`, we will need to create a `Mapping` so the cert-manager can reach the temporary pod.
```

Comment:
This is no longer true. Ambassador added support the `Ingress` since this was originally written.

It might actually be easier to use the `Ingress` cert-manager creates for this. You should take a look at it and see if you can get it to work.

Reply:
For the time being, I think, I know this works for a fact, I did spend some time last week trying to create an Ingress that worked similarly to the mapping and was struggling.  I also know is that the Ingress that is applied out of the box by `cert-manager` does not work.